BookinV2.Data/Entities/BaseEntity.cs

```diff
@@ -0,0 +1,7 @@
+namespace BookinV2.Data.Entities
+{
+    public abstract class BaseEntity
```

Comment:
you dont have to fix it now but you should start developing habit to write metainfo comments on important system parts like classes, public interface methods and so on